src/caduceus/senderWrapper.go

```diff
@@ -163,6 +168,8 @@ func (sw *CaduceusSenderWrapper) Update(list []webhook.W) {
 // function performs the fan-out and filtering to multiple possible endpoints.
 func (sw *CaduceusSenderWrapper) Queue(msg *wrp.Message) {
 	sw.mutex.RLock()
+	sw.eventType.With("event_type", strings.TrimLeft(msg.Destination[23:], "/")).Add(1.0)
```

Comment:
This should ideally be done by WRP library... but until then, we should do it the same way as https://github.com/Comcast/caduceus/blob/master/src/caduceus/outboundSender.go#L514